scarb/src/bin/scarb/args.rs

```diff
@@ -341,6 +342,8 @@ pub struct FmtArgs {
     /// Specify package(s) to format.
     #[command(flatten)]
     pub packages_filter: PackagesFilter,
+    #[clap(value_name = "TARGET")]
+    pub target: Option<PathBuf>,
```

Comment:
I don't like this name. It collides with Scarb Targets.

Wdyt about `--file`?

Reply:
actually what about making this `PATH` and instead formatting the entire path - if that path happens to be a file, only a single file will be formatted?

Reply:
sounds cool